ucs/chatchannel.h

```diff
@@ -69,19 +75,23 @@ class ChatChannel {
 class ChatChannelList {
 
 public:
-	ChatChannel* CreateChannel(std::string Name, std::string Owner, std::string Passwordi, bool Permanent, int MinimumStatus = 0);
+	ChatChannel* CreateChannel(std::string Name, std::string Owner, std::string Password, bool Permanent, int MinimumStatus, bool SaveToDB = false);
```

Comment:
Should these be `const std::string&` to avoid making a copy?

Reply:
`name`, `owner`, `password`, `permanent`, `minimum_status`, and `save_to_database`/`save_to_db`?